tests/microos/install_image.pm

```diff
@@ -29,10 +29,17 @@ sub run {
     # SL Micro x86_64 image has three partitions, aarch64 and ppc64le images have only two partitions
     my $root_partition_id = is_x86_64 ? 3 : 2;
     my $root_partition = "${device}" . $prefix . $root_partition_id;
-    # New partition id for ignition will be directly after root partition
-    my $ignition_partition_id = $root_partition_id + 1;
-    my $ignition_partition = "${device}" . $prefix . $ignition_partition_id;
-    record_info("Device information", "Device: ${device}\nRoot partition: ${root_partition}\nIgnition partition: ${ignition_partition}");
+
+    my $ignition_partition;
+
+    my $msg = "Device: ${device}\nRoot partition: ${root_partition}\n";
```

Comment:
I would put this assignation right before the `record_info` below where you use it.